content/en/tracing/service_catalog/_index.md

```diff
@@ -114,6 +114,16 @@ To access additional details describing security vulnerabilities and signals, cl
 
 Click the Settings icon on the right hand corner to hide metric columns from the service list.
 
+### Software Delivery view
+The **Software Delivery tab** provides several ways to assess and improve the pre-production status of your services. This includes understanding the status of your CI and viewing your static analysis violations. Search for the service you are interested in to:
```

Comment:
**Software Delivery** tab

Reply:
"The status of your CI pipelines"

Reply:
Replace "Search for the service you are interested in" to "You can:"

(When you search you leave to go to the specific service page instead of staying on the Delivery lens)